EKF/control.cpp

```diff
@@ -793,11 +795,14 @@ void Ekf::controlBetaFusion()
 
         }
 
-	// Attempt synthetic sideslip fusion at the prescribed rate if we are in-flight and fusion has been enabled
+	// Attempt synthetic sideslip fusion at the prescribed rate if we are in-flight beta fusion has been enabled externally
+	// and the filter is operating in a normal aiding mode
 	bool beta_fusion_time_triggered = _time_last_imu - _time_last_beta_fuse > _params.beta_avg_ft_us;
-	if(beta_fusion_time_triggered && _control_status.flags.fuse_beta && _control_status.flags.in_air){
+	bool is_aiding = _control_status.flags.gps || _control_status.flags.opt_flow || _control_status.flags.ev_pos;
+	if(beta_fusion_time_triggered && _control_status.flags.fuse_beta && _control_status.flags.in_air && is_aiding) {
 		// If starting wind state estimation, reset the wind states and covariances before fusing any data
 		if (!_control_status.flags.wind) {
+			ECL_INFO("EKF commencing wind estimation using sideslip");
```

Comment:
I get that, but it will eventually make it into the log. Do you want / need that? Otherwise I would remove it.